packages/protocol/test/bridge/libs/LibBridgeData.test.ts

```diff
@@ -2,8 +2,7 @@ import { expect } from "chai";
 import { ethers } from "hardhat";
 import { TestLibBridgeData } from "../../../typechain";
 import { K_BRIDGE_MESSAGE } from "../../constants/messages";
-import { MessageStatus } from "../../../tasks/utils";
-import { Message } from "../../utils/message";
+import { Message, MessageStatus } from "../../utils/message";
```

Comment:
Later one maybe we want to remove all these LIbBridgeXXX.test.ts and treat Bridge as a whole.